dist/ghp-updater.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${ABSOLUTE_PATH}/commons.sh"
+TARGET_DIR="${PROJECT_PARENT_DIR}target"
```

Comment:
Is there a `/` missing or is that included in `PROJECT_PARENT_DIR`? If the latter is true we should probably remove it from there.

Reply:
`+ RELEASE_OUTPUT_DIR=/home/shiywang/ART/cincinnati/dist/..//release-20190621.104710`

Yeah, I will remove it